nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/settings/feature/_settings.component-theme.scss

```diff
@@ -17,18 +17,17 @@
 
 @use 'sass:map';
 @use '@angular/material' as mat;
+@use '../../../../../node_modules/@angular/material/core/theming/inspection' as inspection;
```

Comment:
```suggestion
@use '@angular/material/core/theming/inspection' as inspection;
```

Reply:
Tried this originally, and then again just now, it throws an error `Can't find stylesheet to import.`